src/Illuminate/Validation/Rules/Password.php

```diff
@@ -300,8 +300,6 @@ public function passes($attribute, $value)
                 return;
             }
 
-            $value = (string) $value;
```

Comment:
This cast statement is useless. Because after the first return of the check ! is_string($value), $value is absolutely a string.